lib/card-types.js

```diff
@@ -115,7 +115,12 @@ var cardTypes = {
       [627781, 627799],
       [6282, 6289],
       6291,
-      6292
+      6292,
+      810,
```

Comment:
I think w should be able to do

```
8,
81,
```

as well since it's the only card that starts with 8.

Reply:
Wait, I think we adjusted the parsing to allow 810 to just work for 8. I'll have to check.